xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -590,6 +596,46 @@ void CGUIDialogVideoInfo::ClearCastList()
   m_castList->Clear();
 }
 
+void CGUIDialogVideoInfo::OnChangeVersion()
+{
+  CVideoDatabase videodb;
+  if (!videodb.Open())
+    return;
+
+  CFileItemList list;
+  int dbId = m_movieItem->GetVideoInfoTag()->m_iDbId;
+
+  CGUIDialogSelect* dialog = CServiceBroker::GetGUI()->GetWindowManager().GetWindow<CGUIDialogSelect>(WINDOW_DIALOG_SELECT);
+  if (!dialog)
+    return;
+
+  int selected = -1;
+
+  videodb.GetTypesNav("videodb://movies/types", list, dbId);
+  list.Sort(SortByLabel, SortOrderAscending, CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_FILELISTS_IGNORETHEWHENSORTING) ? SortAttributeIgnoreArticle : SortAttributeNone);
+  dialog->Reset();
+  dialog->SetItems(list);
+  dialog->SetHeading(CVariant{39307});
+  dialog->SetSelected(videodb.GetMovieCurrentVersion(dbId));
+  dialog->Open();
+
+  if (dialog->IsConfirmed())
```

Comment:
I would move `int selected = -1;` above this line, where the var is first needed.